mathematica/mathematica_test.cpp

```diff
@@ -314,6 +276,11 @@ TEST_F(MathematicaTest, PlottableDataset) {
 
 TEST_F(MathematicaTest, Escape) {
   EXPECT_EQ("\"foo\"", Escape("foo"));
+  EXPECT_EQ("\"fo\\\"o\"", Escape("fo\"o"));
+  EXPECT_EQ("\"fo\\\\o\"", Escape("fo\\o"));
+  EXPECT_EQ("\"fo\\no\"", Escape("fo\no"));
+  EXPECT_EQ("\"fo\\to\"", Escape("fo\to"));
+  EXPECT_EQ("\"\"", Escape(""));
```

Comment:
Raw strings for the expected values; https://xkcd.com/1638/.

Reply:
Done (but one of them confuses the macro).